files/zh-cn/web/javascript/reference/global_objects/string/towellformed/index.md

```diff
@@ -15,26 +15,26 @@ toWellFormed()
 
 ### 返回值
 
-新的字符串是原字符串的一个拷贝，其中所有的单独的代理项被替换为 Unicode 替换字符 U+FFFD。如果 `str` [是格式正确的](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/String/isWellFormed)，仍然会返回一个新字符串（本质上是 `str` 的一个拷贝）。
+新的字符串是原字符串的一个拷贝，其中所有的单独代理项被替换为 Unicode 替换字符 U+FFFD。如果 `str` [是格式正确的](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/String/isWellFormed)，仍然会返回一个新字符串（本质上是 `str` 的一个拷贝）。
 
 ## 描述
 
-JavaScript 中的字符串是 UTF-16 编码的。UTF-16 编码中有*代理对*的概念，这一概念在 [UTF-16 字符、Unicode 码位和字素簇（grapheme clusters）](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/String#utf-16_字符、unicode_码位和字素簇（grapheme_clusters）)部分有详细介绍。
+JavaScript 中的字符串是 UTF-16 编码的。UTF-16 编码中有*代理对*的概念，这一概念在 [UTF-16 字符、Unicode 码位和字素簇](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/String#utf-16_字符、unicode_码位和字素簇)部分有详细介绍。
 
-`toWellFormed()` 迭代字符串的码元，并将任何单独的代理项替换为 [Unicode 替换字符](<https://zh.wikipedia.org/wiki/特殊_(Unicode區段)>) U+FFFD `�`。这确保了返回的字符串格式正确并可用于期望正确格式字符串的函数，比如 {{jsxref("encodeURI")}}。由于引擎能够直接访问字符串的内部表示，与自定义实现相比 `toWellFormed()` 更高效。
+`toWellFormed()` 迭代字符串的码元，并将任何单独代理项替换为 [Unicode 替换字符](<https://zh.wikipedia.org/wiki/特殊_(Unicode區段)>) U+FFFD `�`。这确保了返回的字符串格式正确并可用于期望正确格式字符串的函数，比如 {{jsxref("encodeURI")}}。由于引擎能够直接访问字符串的内部表示，与自定义实现相比 `toWellFormed()` 更高效。
```

Comment:
The `\U+FFFD` Unicode character makes the gh command fail: cli/go-gh#127. I will review the PR locally. Do not worry about the failed CIs.